relayer/src/services/relayer/arbitrum.ts

```diff
@@ -145,24 +226,40 @@ export class ArbitrumRelayerService extends BaseRelayerService {
     const wrappedL1TransactionReceipt = new L1TransactionReceipt(l1TransactionReceipt);
     const messages = await wrappedL1TransactionReceipt.getL1ToL2Messages(this.l2Wallet);
     const message = messages[0];
+    return message;
+  }
+
+  private async isFinalizeCompleted(l1TransactionHash: string): Promise<void | false> {
+    if (!this.l1ToL2Message) {
+      this.l1ToL2Message = await this.getL1ToL2Message(l1TransactionHash);
+    }
 
     console.log("Waiting for L2 side. It may take 10-15 minutes ⏰⏰");
-    const messageResult = await message.waitForStatus();
+    const messageResult = await this.l1ToL2Message.waitForStatus();
 
     const status = messageResult.status;
     if (status === L1ToL2MessageStatus.REDEEMED) {
       console.log(`L2 retryable txn executed ${messageResult.l2TxReceipt.transactionHash}`);
       return;
```

Comment:
`return false;` instead?

Reply:
If it is redeemed, the step is completed and we return the default return value of the step (`void`).

Reply:
aah, true. My bad